src/CarbonAware.WebApi/src/Models/EmissionsDataDTO.cs

```diff
@@ -22,7 +21,7 @@ public record EmissionsDataDTO
     [JsonPropertyName("value")]
     public double Value { get; set; }
 
-    public static EmissionsDataDTO? FromEmissionsData(EmissionsData emissionsData)
+    public static EmissionsDataDTO? FromEmissionsData(global::CarbonAware.Model.EmissionsData emissionsData)
```

Comment:
Seems like this method is not longer needed.

Reply:
Removed